tutorials/10_DebuggingREMIND.md

```diff
@@ -0,0 +1,183 @@
+
+If you only want to look at a specific region that created the infeasibility, you can either specify `c_testOneRegi_region` in `scenario_config_XYZ.csv`:
+```bash
+Rscript start.R --debug --interactive --testOneRegi scenario_config_XYZ.csv
+```
+and you will be asked which region you want to look at (the shortcut is `Rscript start.R -di1` + the file name).
+You can also run `Rscript start.R --reprepare --debug --testOneRegi` to get the results in the same output folder.
+Running `Rscript start.R --reprepare` a second time then resets the settings to the original ones.
+
+If the error was not in the first iteration, a faster and more convenient way might be to start the run again in parallel mode with `c_keep_iteration_gdxes = 1` specified in `config/default.cfg` or the `scenario_config_XYZ.csv`. And then start a debug run that uses the `fulldata_07.gdx` (or the first iteration in which this occurs) of this first run as `input.gdx` specified in `path_gdx` of `scenario_config_XYZ.csv`. That way, you should see where the infeasibility is in the first iteration of the debug run already and not have to wait until the 7th iteration.
+
+If the iterations are not sufficient to converge, you can run REMIND for more iterations by modifying `cm_iteration_max` in [`80_optimization/nash/datainput.gms`](../modules/80_optimization/nash/datainput.gms) and the set iteration in [`core/sets.gms`](../core/sets.gms), which by default only goes to 200.
+
+If you want a closer look on the GAMS CONOPT output during a REMIND run, you can access the solver logs by moving to the output folder and running:
+
+```bash
+find -name gmsgrid.log | xargs tail -n 12
+```
+Users of the PIK cluster can use `conoptspy` instead.
+
+Only the first five columns are of particular interest:
+
+- `Iter`: CONOPT Iteration number.
+- `Phase`: 0/1/2 means CONOPT is looking for a feasible solution, 3/4 means it's looking for an optimal solution.
+- `Ninf`: if during phase 1/2, the number of infeasibilities.
+- `Infeasibility/Objective`: The left hand side during search for a feasible solution, and the objective value during phase 3/4.
+- `RGmax`: if during phase 1-2 the sum of infeasibilities, it should always converge to a very small value (< 10e-7). If during phase 3/4 the reduced gradient (when small you are close to optimality).
+- `NSB`: the number of superbasic variables (basically the current number of degrees of freedom).
+
+Asking for help
+----------------------------------
+
+In any case, don't hesitate to ask for help in [pik-piam/discussions](https://github.com/pik-piam/discussions). Please provide:
+
+- as much information on your run, such as the scenario config file and the run name
+- PIK cluster users should supply the path on the cluster
+- a summary of things you already checked or found out
+- information on what you changed in the REMIND GAMS code
```

Comment:
add: ...based on which version of REMIND

Reply:
I did